skypy/utils/photometry.py

```diff
@@ -392,3 +394,53 @@ def magnitude_error_rykoff(magnitude, magnitude_limit, magnitude_zp, a, b, error
     error = 2.5 / np.log(10) * np.sqrt((1 + flux_noise / flux) / (flux * t_eff))
 
     return np.minimum(error, error_limit)
+
+
+def logistic_completeness_function(magnitude, magnitude_95, magnitude_50):
+    r'''Compute logistic completeness function.
```

Comment:
This should be more of a title, either
```suggestion
    r'''Logistic completeness function.
```

or
```suggestion
    r'''Computation of the logistic completeness function.
```

Reply:
I used your first suggestion and changed the docstring.

I guess now we need to see if @rrjbca agrees with the changes.